config/routes.rb

```diff
@@ -3,6 +3,8 @@
 
   resources :articles do
     resources :comments, only: [:create, :destroy]
+    resources :likes, only: [:create, :destroy]
+    delete '/unlike' => 'likes#destroy'
```

Comment:
I try, but default destroy action require like_id, and this param useless in my action